src/Core/Framework/Framework.php

```diff
@@ -152,16 +149,6 @@ public function boot(): void
             MeterProvider::bindMeter($this->container);
         }
 
-        // @deprecated tag:v6.7.0 - remove complete if condition
-        if (!Feature::isActive('DYNAMIC_ENTITY_EXTENSIONS')) {
```

Comment:
as the flag is unused now (and was only added temporarily) we should remove it from the feature.yaml file as well